src/Symfony/Component/DependencyInjection/Exception/ContextElementNotFoundException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Exception;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ContextElementNotFoundException extends InvalidArgumentException
```

Comment:
Lets keep it. This is hard to explain (my pov) for PHP; everything is runtime, and everything is a programmers fault too eventually. I went down that road long time ago, https://stackoverflow.com/questions/14171714/php-runtime-or-logic-exception